Dockerfile

```diff
@@ -66,6 +66,7 @@ ARG VARIANT_RUNTIME
 ARG VARIANT_FAMILY
 ARG VARIANT_FLAVOR
 ARG REPO
+ARG GRUB_SET_PRIVATE_VAR
```

Comment:
Elsewhere I have noted that these are effectively an enum of allowable build args.